packages/opencensus-core/package.json

```diff
@@ -6,15 +6,14 @@
   "types": "build/src/index.d.ts",
   "repository": "census-instrumentation/opencensus-node",
   "scripts": {
-    "test": "nyc mocha build/test/**/*.js",
+    "test": "nyc ts-mocha -p ./tsconfig.json test/**/*.ts",
     "codecov": "nyc report --reporter=json && codecov -f coverage/*.json",
     "clean": "rimraf build/*",
     "check": "gts check",
     "compile": "tsc -p .",
     "compile:release": "tsc -p tsconfig-release.json",
     "fix": "gts fix",
     "prepare": "npm run compile:release",
-    "pretest": "npm run compile",
```

Comment:
I think we should still keep this and add into unit test workflow. Something like below in `.circleci/config.yml` WDYT?
```
   - run:
        name: Compile code
        command: npm run compile
```

Reply:
It looks like that config is for the entire repo.  That would cause all of the other packages to run the compile step twice.  Is that ok and/or should I change all of the other packages to use ts-mocha?

Reply:
I would prefer to change all packages to use `ts-mocha`. Also, does it make sense to remove dependencies on `mocha` and `@types/mocha` after this? @draffensperger any thoughts?

Reply:
No, we can't remove the dependencies.  The ts-mocha page says we need them: https://www.npmjs.com/package/ts-mocha

Reply:
Ok SG.